responsive-images/styles2-images.css

```diff
@@ -0,0 +1,37 @@
+/* responsive images */
+.images-container {
+  margin: 3rem;
+}
+
+.images-main-heading {
+  color: yellow;
+}
+.images-snippet iframe {
+  width: 564px;
+  height: 256px;
+  border: 0;
+  transform: scale(1);
+  overflow: hidden;
+}
+.images-sub {
+  margin: 1rem 0rem;
+  color: white;
+  font-size: 1.2rem;
+}
+.images-snippet {
+  margin: 1rem 0rem;
+}
+.standard-images {
+  color: yellow;
```

Comment:
This can give different render values on different browsers, use hex code and then covert it into var

Reply:
done